### DISCUSSION TITLE
problem after build web

### DISCUSSION THREAD

**Discussion by OtherwisePitt92** [flet-dev/flet#4072] [General]
Probably i'm making some simple mistake but after build my webapp, i upload it on cloudfare but it's give my a blank page.
i thnk the problem is not pyre python package. for build webapp. how i can manage these package like supabase?

**Comment by ndonkoHenri** [comment]
When using flet build web, not all python packages can be packaged. More info here: https://flet.dev/docs/publish/web

If you make use of unsupported packages, then I will suggest you deploy your web app as [dynamic](https://flet.dev/docs/publish/web#:~:text=Any%20Python%20package%20can%20be%20used.) website.

**Reply by OtherwisePitt92** [reply]
i love your video. make a video for  deploy  web app as dynamic website

**Reply by OtherwisePitt92** [reply]
but i cannot use micropip or something to install pagkages like sqllite3 and supabase?
